tests/unit/components/App/LocaleSwitcher.spec.js

```diff
@@ -0,0 +1,18 @@
+import { mount } from "@vue/test-utils";
+import LocaleSwitcher from "@/components/App/LocaleSwitcher";
+
+describe("LocaleSwitcher.vue", () => {
+  it("should render with default values", () => {
+    const wrapper = mount(LocaleSwitcher);
+    expect(wrapper).toBeTruthy();
+  });
+
+  it("test updating switcher triggers the update method", () => {
```

Comment:
```suggestion
  it("updating switcher triggers the update method", () => {
```
"test" keyword is not required here right? looks much cleaner without it